### ISSUE TITLE
does this work with vue.js?

### ISSUE THREAD

**Issue by ralyodio** [joelmukuthu/animated-scroll#21]
The promise callback on `scroll.left(offset).then()` never fires for me inside a vue.js component.

```
    prev(id) {
      const el = document.getElementById(id);
      const scroll = new AnimatedScroll(el);
      // el.scrollLeft -= el.offsetWidth;

      scroll.left(el.offsetWidth).then((newLeft) => {
        // newLeft === 100
        console.log('#myElement\'s scrollLeft is now', newLeft);
      });
    },
```

**Comment by joelmukuthu** [comment]
Frankly I never got round to finishing this project, hence the low coverage. Taking a quick look at the code, it looks like there's a bug here: 
https://github.com/joelmukuthu/animated-scroll/blob/3c886ba65efb4fc0c66a3e648d33b33436492ac0/src/AnimatedScroll.js#L79

I'm guessing that should be `leftAnimation`. Could you try fixing that in your environment and see if it works? If you're using the transpiled code then the fix would be here:
https://github.com/joelmukuthu/animated-scroll/blob/3c886ba65efb4fc0c66a3e648d33b33436492ac0/dist/AnimatedScroll.js#L178

**Comment by Zauberfisch** [comment]
Works with vuejs. Main reason it might not work is probably #40.

`animationProperty = 'topAnimation';` instead of `animationProperty = 'leftAnimation';` only matters if you animate both `top` and `left` I think.
Though this is an easy fix, @joelmukuthu mind quickly replacing that before looking at #40?

**Comment by Zauberfisch** [comment]
I've created a PR for the `animationProperty`. From reading the source again, the only thing this bug effected was stopping. So stopLeft() would not be able to cancel animation frame.

Though what OP reported (not working with vuejs) is probably not related to animationProperty but rather just the broken build (#40). Using the file in `src/` works in vuejs. Therefore I believe this issue can be closed.

**Comment by joelmukuthu** [comment]
Thank you for the findings @Zauberfisch!

**Comment by joelmukuthu** [comment]
FYI I published v0.0.3 which should fix the build and import.

**Comment by Zauberfisch** [comment]
Can confirm, v0.0.3 works with vuejs
